### PR TITLE
chore: linting and cleanup

### PR DESCRIPTION
**Pull request by hairmare**
This is a first step towards #69 (haha nice 😏).

A lot of it is faffing about with basics like pylint.

It's the start of some more proactive maintaining of `suisa_sendemeldung` and also aligning it with the foundational stuff i've done in `nowplaying` et al. over the last few weeks/months.

### CODE REVIEW

**Comment by hairmare** [comment]
@spameier lemme know if this is ok, i have some more changes in the works for `suisa_sendemeldung` but it's gonna be slow evolution given how rarely this things runs in prod.

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (spameier) [inline .vscode/settings.json:1]: Would it be possible to make this setting using EditorConfig?

**Review by spameier — CHANGES_REQUESTED** [review]
Thanks for your effort! Lookgs good to me except the EditorConfig thingy.

File: .vscode/settings.json (hunk 1 above)
Comment (hairmare) [reply .vscode/settings.json]: It needs an [additional extension](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig) in VS Code but i'm ok with that and have updated the PR

**Review by spameier — APPROVED** [review]

**Pull request merged**